.github/workflows/check-rtw.yml

```diff
@@ -33,19 +33,13 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job
       # can access it
-      - uses: actions/checkout@v4
-      - uses: conda-incubator/setup-miniconda@v3
-        with:
-          miniforge-version: "latest"
-          miniforge-variant: Miniforge3
-          use-mamba: true
-          conda-remove-defaults: "true"
-
-      - name: Install Cylc and Rose
-        run: conda install cylc-flow>=8.2 cylc-rose metomi-rose
+      - name: Checkout repository
+        uses: actions/checkout@v4
 
-      - name: Check current environment
-        run: conda list
```

Comment:
I removed this because I hope the output from the new "Setup Cylc" Action (which I learned about yesterday!) is sufficient 😊

Reply:
that's a great find, I didn't know about that one

Reply:
Me neither! 🤣